compiler/plugins/target/ROCM/test/default_tuning_specs_amdgpu.mlir

```diff
@@ -33,11 +33,11 @@
 // Check that both the user tuning spec and the default spec get linked and
 // materialized. The user spec should have precedence over the default one.
 
-// BOTH-LABEL: module @iree_linked_tuning_spec attributes {transform.with_named_sequence}
+// BOTH-LABEL: module @iree_linked_tuning_spec attributes {iree_codegen.tuning_spec_with_default_entrypoint, transform.with_named_sequence}
```

Comment:
Can you use `BOTH-SAME` to make this match work for any ordering of these module-level attributes?